pages/JSplayground.js

```diff
@@ -0,0 +1,14 @@
+export default function Home() {
+  return (
+    <div className="h-full ">
+      <h1 className="text-2xl md:text-7xl flex justify-center ">Welcome to JavaScript playground</h1>
```

Comment:
We have `JS` and `React` playgrounds. I think we can make a reusable component to render the `header` and playground accordingly.